static/umap/theme.css

```diff
@@ -0,0 +1,76 @@
+/* **************************** */
+/*           navigation         */
+/* **************************** */
+nav h1 a {
+    background-image: url('./img/logo-marianne.gif');
+}
+.login-grid .login-datagouv {
+    background-image: url('../datagouv.png');
+}
+.branding {
+    width: 33%;
+}
+
+
+/* **************************** */
+/*             Forms            */
+/* **************************** */
+input[type="submit"], .button {
+    background-color: #2185d0;
+    color: #eeeeec;
+    padding-left: 20px;
+    padding-right: 20px;
+    font-family: 'fira_sansregular';
```

Comment:
No fallback? See what I did [for the blog proposal](https://github.com/davidbgk/blog2.data.gouv.fr/blob/master/basic_theme/static/css/style.css#L2)